WordPress/wordpress-playground
# Try memory leak workaround with zeroed mem

## What is this PR doing?

This PR attempts to fix the memory leak reported in #1128 and is an iteration on PR #1189 which had problems with "memory out of bounds" errors.

## What problem is it solving?

It stops PHP from leaking memory throughout the runtime of a script and hopefully stops memory out of bounds errors by zeroing all memory given to PHP.

## How is the problem addressed?

- By avoiding mmap()/munmap() which have incomplete implementations in Emscripten
- By using posix_memalign() to allocate memory instead and manually zeroing the memory before handing it to PHP

## Testing Instructions

- Observe CI test results
- Use `npm run dev` and exercise Playground locally in the browser
